test/test.js

```diff
@@ -0,0 +1,43 @@
+var assert = require('chai').assert;
+var index = require('../index.js');
+var app = require('../app.js');
+var request = require('request');
+const https = require('https');
+
+describe('HTTPS response', function () {
+    describe('', function () {
```

Comment:
Generally, I wouldn't use nested `describe` blocks unless you want to add a more specific message for a sub-group of tests.